src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.Properties.cs

```diff
@@ -0,0 +1,477 @@
+
+	#region DependencyProperty: VerticalZoomCenter
+
+	/// Identifies the VerticalZoomCenter dependency property.
+	public static DependencyProperty VerticalZoomCenterProperty { get; } = DependencyProperty.Register(
+		nameof(VerticalZoomCenter),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double)));
+
+	/// Gets or sets the vertical center point for zooming.
+	public double VerticalZoomCenter
+	{
+		get => (double)GetValue(VerticalZoomCenterProperty);
+		set => SetValue(VerticalZoomCenterProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: ViewportWidth
+
+	/// Identifies the ViewportWidth dependency property.
+	public static DependencyProperty ViewportWidthProperty { get; } = DependencyProperty.Register(
+		nameof(ViewportWidth),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double)));
+
+	/// Gets or sets the width of the viewport.
+	public double ViewportWidth
+	{
+		get => (double)GetValue(ViewportWidthProperty);
+		set => SetValue(ViewportWidthProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: ViewportHeight
+
+	/// Identifies the ViewportHeight dependency property.
+	public static DependencyProperty ViewportHeightProperty { get; } = DependencyProperty.Register(
+		nameof(ViewportHeight),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double)));
+
+	/// Gets or sets the height of the viewport.
+	public double ViewportHeight
+	{
+		get => (double)GetValue(ViewportHeightProperty);
+		set => SetValue(ViewportHeightProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: ZoomLevel
+
+	/// Identifies the ZoomLevel dependency property.
+	public static DependencyProperty ZoomLevelProperty { get; } = DependencyProperty.Register(
+		nameof(ZoomLevel),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(1d, OnZoomLevelChanged));
+
+	/// Gets or sets the current zoom level.
+	public double ZoomLevel
+	{
+		get => (double)GetValue(ZoomLevelProperty);
+		set => SetValue(ZoomLevelProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: MinZoomLevel
+
+	/// Identifies the MinZoomLevel dependency property.
+	public static DependencyProperty MinZoomLevelProperty { get; } = DependencyProperty.Register(
+		nameof(MinZoomLevel),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double), OnMinZoomLevelChanged));
+
+	/// Gets or sets the minimum zoom level allowed.
+	public double MinZoomLevel
+	{
+		get => (double)GetValue(MinZoomLevelProperty);
+		set => SetValue(MinZoomLevelProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: MaxZoomLevel
+
+	/// Identifies the MaxZoomLevel dependency property.
+	public static DependencyProperty MaxZoomLevelProperty { get; } = DependencyProperty.Register(
+		nameof(MaxZoomLevel),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(500d, OnMaxZoomLevelChanged));
```

Comment:
That's quite too large. Was this default value discussed with @Xiaoy312 ?

Reply:
agreed, 500x would be a bit excessive. 10x would probably make more sence here: 
```suggestion
		new PropertyMetadata(10d, OnMaxZoomLevelChanged));
```